blocks/rich-text/index.js

```diff
@@ -756,7 +740,14 @@ export class RichText extends Component {
 			this.props.value !== prevProps.value &&
 			this.props.value !== this.savedContent
 		) {
-			this.updateContent();
```

Comment:
Curious why this was pulled out of the function and made inline. Ideally we should favor clearly named functions over inline logic in component lifecycle.

Reply:
Because at the start of this PR it was just one line and then piled up. :) Will move it back to a function.